<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>455</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>811</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>About OpenShot</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../images/openshot.svg</normaloff>../../images/openshot.svg</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap">
        <pixmap>../../images/AboutLogo.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txtversion">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 4em; font-weight: bold;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblAboutDescription">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblAboutCompany">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btncredit">
         <property name="text">
          <string>Credits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnlicense">
         <property name="text">
          <string>License</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnchangelog">
         <property name="text">
          <string>Changelog</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>777</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>765</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
